plone/app/contenttypes/configure.zcml

```diff
@@ -8,6 +8,7 @@
            i18n_domain="plone.app.contenttypes">
 
   <includeDependencies package="." />
+  <include package="plone.app.event" />
```

Comment:
That's a redundancy - the includeDependencies node above already imports plone.app.event via z3c.autoinclude.
The only problem is, that in the (plone fixture IIRC) test layer z3c.autoinclude is disabled, so you have to explicitly load it.
You probably should use the ``plone.app.contenttypes.testing.
PLONE_APP_CONTENTTYPES_FIXTURE`` as a base in your own test layer, no?